element-hq/element-x-android
# Sort the room member list and display member roles

## Type of change


## Content

- Sorts the room member list by power level (descending) first, then by name (ascending).
- Adds role to the room member items if it's not a base user one.

## Motivation and context

First part of #2256.

## Screenshots / GIFs

In the PR.

## Tests



- Open a room, then its details and member list.
- Make sure the joined users are sorted Admin > Moderator > User, and internally they're sorted alphabetically.

## Tested devices

- OS version(s): 14

## Checklist
